app/views/me/notices/index.html.haml

```diff
@@ -1,12 +1,24 @@
-.h2.first=:notices.t
+%div
+  %span.h2.first=:notices.t
+  - if @notices.total_pages < 2 && @notices.any?
+    %span.h2.dissmiss-all-notices-button.dismiss-all-wrapper.pull-right
```

Comment:
Oh, my fault, have committed into git ignored file! :D

Let me fix this and other typo issues a bit later.
On Jul 11, 2015 5:35 PM, "azul" <notifications@github.com> wrote:

> In app/views/me/notices/index.html.haml
> <https://github.com/riseuplabs/crabgrass-core/pull/269#discussion_r34413164>
> :
>
> > @@ -1,12 +1,24 @@
> > -.h2.first=:notices.t
> > +%div
> > +  %span.h2.first=:notices.t
> > +  - if @notices.total_pages < 2 && @notices.any?
> > +    %span.h2.dissmiss-all-notices-button.dismiss-all-wrapper.pull-right
>
> hmm... i searched the diff page on github for
> "dissmiss-all-notices-button" and i only found this one occurance. Am i
> missing something?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/riseuplabs/crabgrass-core/pull/269/files#r34413164>.
>